stingray/gti.py

```diff
@@ -1020,6 +1079,51 @@ def time_intervals_from_gtis(gtis, chunk_length, fraction_step=1,
     return spectrum_start_times, spectrum_start_times + chunk_length
 
 
+def get_segment_bin_start(startbin, stopbin, nbin, fraction_step=1):
+    """Get start of intervals of equal length, given bin number range
+
+    Parameters
+    ----------
+    startbin : int
+        Starting bin of the interval
+
+    stopbin : int
+        Last bin of the interval
+
+    nbin : int
+        number of bins in each interval
+
+    Other Parameters
+    ----------------
+    fraction_step : float
+        If the step is not a full ``chunk_length`` but less (e.g. a moving window),
```

Comment:
Is ``chunk_length`` a parameter of this function? Of another function? It's not really defined here?

Reply:
nbin. Corrected